tap_stripe/__init__.py

```diff
@@ -431,6 +423,19 @@ def get_bookmark_for_stream(stream_name, replication_key):
             int(utils.strptime_to_utc(Context.config["start_date"]).timestamp())
     return stream_bookmark
 
+def get_bookmark_for_sub_stream(stream_name):
+    """
+    Get the bookmark for the child-stream based on the parent's replication key.
+    """
+    child_stream = stream_name
+    # Get the parent for the stream
+    parent_stream = PARENT_STREAM_MAP[child_stream]
+    # Get the replication key
+    replication_key = STREAM_REPLICATION_KEY[parent_stream]
```

Comment:
Is it a child's bookmark key?

Reply:
No currently in the tap, its using the parent's bookmark key to fetch the bookmark from the state for the child

Reply:
```suggestion
    parent_replication_key = STREAM_REPLICATION_KEY[parent_stream]
```